bottlecap/src/bin/bottlecap/main.rs

```diff
@@ -161,10 +163,16 @@ fn main() -> Result<()> {
 
     let logs_agent = LogsAgent::run(&function_arn, Arc::clone(&config));
     let event_bus = EventBus::run();
+    let mut metadata_hash = hash_map::HashMap::new();
+    metadata_hash.insert("function_arn".to_string(), function_arn.clone());
```

Comment:
Can't we just initialize it with the data?

Reply:
TIL hashmap::from() was introduced in 1.56